pom.xml

```diff
@@ -30,7 +30,9 @@
 
   <properties>
     <stack.version>4.2.0-SNAPSHOT</stack.version>
-    <kafka.version>2.6.0</kafka.version>
+    <kafka.version>2.7.0</kafka.version>
```

Comment:
Kafka version was changed due to DescribeFeatureResult being unavailable in previous version